### PR TITLE
Update script link margin

### LINKED ISSUE #5062: Padding missing below script link on small screen widths
**This is part of the navigation integration work, and should be based off the integrate-new-navigation feature branch.**

**Describe the bug**
The padding below the script link is missing on small screen widths.
See designs in zeplin:

https://app.zeplin.io/project/5c8a5e021d454124ef90669f/screen/5db96f04b9175d22aa0036c5

**To Reproduce**

1. Go to http://localhost:7080/serbian/lat
2. Reduce screen width below 400px.
4. See error:

Screenshot 2020-01-08 at 14 51 22

**Expected behavior**
The correct padding should be observed.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by AlistairGempf**
Resolves https://github.com/bbc/simorgh/issues/5062

**Overall change:** Update margins on the script link to ensure 8px margin between it and brand svg.

**Code changes:**

- Add 0.5rem margin on left, top and bottom, 0rem margin to the right, of the script-link.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by AlistairGempf** [comment]
Blocked on #2887

**Review by OlgaLyubin — APPROVED** [review]
Looks good

File: packages/components/psammead-script-link/src/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -13,7 +13,8 @@ exports[`ScriptLink should render correctly 1`] = `
   text-decoration: none;
   height: 2.25rem;
   border: 0.0625rem solid #FFFFFF;
-  margin: 0.25rem;
+  margin: 0.5rem;
```
Comment (DenisHdz) [inline packages/components/psammead-script-link/src/__snapshots__/index.test.jsx.snap]:
```suggestion
  margin: 0.5rem 0 0.5rem 0.5rem;
```

**Review by DenisHdz — APPROVED** [review]
Other than the suggestion it looks good.

**Pull request merged**